src/components/Contacts/ContactDetails/ContactDetailsTab/EditContactModal/EditContactModal.tsx

```diff
@@ -0,0 +1,696 @@
+  position: 'relative',
+  padding: theme.spacing(0, 6),
+  margin: theme.spacing(2, 0),
+}));
+
+// Icon Styles
+const ContactAvatar = styled(Avatar)(() => ({
+  position: 'absolute',
+  top: '50%',
+  left: 4,
+  transform: 'translateY(-50%)',
+  width: '34px',
+  height: '34px',
+}));
+
+const PrimaryContactIcon = styled(BookmarkIcon)(({ theme }) => ({
+  position: 'absolute',
+  top: '50%',
+  left: 8,
+  transform: 'translateY(-50%)',
+  color: theme.palette.cruGrayMedium.main,
+}));
+
+const ContactAddIcon = styled(AddIcon)(() => ({
+  color: '#2196F3',
+}));
+
+const ContactBirthdayIcon = styled(CakeIcon)(({ theme }) => ({
+  position: 'absolute',
+  top: '50%',
+  left: 8,
+  transform: 'translateY(-50%)',
+  color: theme.palette.cruGrayMedium.main,
+}));
+
+const ContactAnniversaryIcon = styled(RingIcon)(({ theme }) => ({
+  position: 'absolute',
+  top: '50%',
+  left: 8,
+  transform: 'translateY(-50%)',
+  color: theme.palette.cruGrayMedium.main,
+}));
+
+const ContactEducationIcon = styled(SchoolIcon)(({ theme }) => ({
+  position: 'absolute',
+  top: '50%',
+  left: 8,
+  transform: 'translateY(-50%)',
+  color: theme.palette.cruGrayMedium.main,
+}));
+
+const ContactJobIcon = styled(BusinessIcon)(({ theme }) => ({
+  position: 'absolute',
+  top: '50%',
+  left: 8,
+  transform: 'translateY(-50%)',
+  color: theme.palette.cruGrayMedium.main,
+}));
+
+const ContactSocialIcon = styled(SocialIcon)(({ theme }) => ({
+  position: 'absolute',
+  top: '50%',
+  left: 8,
+  transform: 'translateY(-50%)',
+  color: theme.palette.cruGrayMedium.main,
+}));
+
+// Input Styles
+const ContactInputField = styled(TextField)(() => ({
+  '&& > label': {
+    textTransform: 'uppercase',
+  },
+}));
+
+// Text Styles
+const ContactEditDeleteIconButton = styled(IconButton)(({ theme }) => ({
+  position: 'absolute',
+  top: '50%',
+  right: 0,
+  transform: 'translateY(-50%)',
+  color: theme.palette.cruGrayMedium.main,
+}));
+
+const ContactPersonNameText = styled(Typography)(() => ({
+  fontWeight: 'bold',
+}));
+
+const ContactAddText = styled(Typography)(() => ({
+  color: '#2196F3',
+  textTransform: 'uppercase',
+  fontWeight: 'bold',
+}));
+
+const ContactPrimaryPersonSelectLabel = styled(InputLabel)(() => ({
+  textTransform: 'uppercase',
+}));
+
+const ShowExtraText = styled(Typography)(() => ({
+  color: '#2196F3',
+  textTransform: 'uppercase',
+  fontWeight: 'bold',
+}));
+
+interface EditContactModalProps {
+  contact: ContactDetailsTabQuery['contact'];
+  isOpen: boolean;
+  handleOpenModal: (open: boolean) => void;
+}
+export const EditContactModal: React.FC<EditContactModalProps> = ({
+  contact,
+  isOpen,
+  handleOpenModal,
+}): ReactElement<EditContactModalProps> => {
+  const { t } = useTranslation();
+  console.log(contact);
+  const [personEditShowMore, setPersonEditShowMore] = useState<Array<string>>(
+    [],
+  );
+
+  const handleDateChange = (date: DateTime) => {
+    console.log(date.month);
```

Comment:
Are you keeping the console.log for a while just while it's in flux?

Reply:
Yeah that's just to get the types to stop yelling since it requires an ```onChange``` function.